server/config.ts

```diff
@@ -120,6 +120,12 @@ export default {
           completedCases: get('PP_MY_CASES_PAGE_SIZE', '500'),
           cancelledCases: get('PP_CANCELLED_CASES_PAGE_SIZE', '500'),
         },
+        sp: {
```

Comment:
_not a blocker_
💭 `config.apis.interventionsService.dashboardPageSize.sp.openCases`

I think I see why this is under API config, however, it can also be considered as UI behaviour -- so it might be worth simplifying this to `config.dashboards.serviceProvider.openCases` or similar

Reply:
agree with this

Reply:
Done